doc/make_cite_html.py

```diff
@@ -0,0 +1,123 @@
+import requests
```

Comment:
copyright header + short description of what the script does and how it needs to be called

Reply:
Do you want to add a copyright header and how the script is called?

Reply:
Yes, I was hoping that @tjhei adds the usual copyright header.